vscode-dotnet-sdk-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -233,19 +235,30 @@ suite('DotnetCoreAcquisitionExtension End to End', function() {
 
     // 5.0 and 3.1 SDKs should still be installed
     sdkDirs = fs.readdirSync(path.join(path.dirname(result!.dotnetPath), 'sdk'));
-    assert.isNotEmpty(sdkDirs.filter(dir => dir.includes('3.1')));
+    assert.isNotEmpty(sdkDirs.filter(dir => dir.includes(currentSDKVersion)));
     assert.isNotEmpty(sdkDirs.filter(dir => dir.includes('5.0')));
 
     // Clean up storage
     await vscode.commands.executeCommand('dotnet-sdk.uninstallAll');
   }).timeout(600000);
 
   test('Extension Uninstall Removes SDKs', async () => {
-    const context: IDotnetAcquireContext = { version: '5.0', requestingExtensionId: 'ms-dotnettools.sample-extension' };
+    const context: IDotnetAcquireContext = { version: currentSDKVersion, requestingExtensionId: 'ms-dotnettools.sample-extension' };
     const result = await vscode.commands.executeCommand<IDotnetAcquireResult>('dotnet-sdk.acquire', context);
     assert.exists(result);
     assert.exists(result!.dotnetPath);
     uninstallSDKExtension();
     assert.isFalse(fs.existsSync(result!.dotnetPath));
   }).timeout(100000);
+
+  test('Extension installs latest SDK version', async () => {
```

Comment:
Why is this new test necessary?

Reply:
I think I added it to have something I could debug easier, but that's just a guess. I think there might even be a bug in this test because it's hard-coding 6.0.201 when the 'currentSDKVersion' might float.